config/rbac/leader_election_role.yaml

```diff
@@ -3,38 +3,39 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   labels:
+    app.kubernetes.io/component: controller
     app.kubernetes.io/name: argocd-ephemeral-access
     app.kubernetes.io/managed-by: kustomize
   name: leader-election-role
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - get
-  - list
-  - watch
-  - create
-  - update
-  - patch
-  - delete
-- apiGroups:
-  - coordination.k8s.io
-  resources:
-  - leases
-  verbs:
-  - get
-  - list
-  - watch
-  - create
-  - update
-  - patch
-  - delete
-- apiGroups:
-  - ""
-  resources:
-  - events
-  verbs:
-  - create
-  - patch
+  - apiGroups:
+      - ''
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
```

Comment:
Do you mind configuring your IDE to stop adding those additional spaces in the yaml? I personally prefer the original format as we end up we less indentation and the yaml is a bit more readable.

Reply:
Prettier does not seem to support this kind of configuration. Which tool are you using to format yaml?

Reply:
Redhat yaml language server provides all sorts of configurations:
https://github.com/redhat-developer/yaml-language-server

Neovim can use language-servers for any language. Maybe VSCode can do it too..